pkg/api/v1/types.go

```diff
@@ -582,6 +582,8 @@ type RBDVolumeSource struct {
 	// Defaults to false.
 	// More info: http://releases.k8s.io/HEAD/examples/rbd/README.md#how-to-use-it
 	ReadOnly bool `json:"readOnly,omitempty"`
+	// Optional: Sidecar is the sidecar container's name
+	Sidecar string `json:"sidecar,omitempty"`
```

Comment:
i expect image name vary by volume. I use `ceph/base` in this example. This image is maintained by ceph community. There could also be a `glusterfs` image to do glusterfs mount, and yet another image for e.g. `fibre channel/iSCSI` related exec.